### ISSUE TITLE
4.1.2 Upgrade: Chef-client hangs on recreate admin user before haproxy listen address is updated

### ISSUE THREAD

**Issue by kstev** [rcbops/chef-cookbooks#931]
While upgrading from 4.1.2 to 4.1.5, chef-client runs fail with the following error:

```
Error executing action `create` on resource 'keystone_tenant[Create 'admin' Tenant]'
```

Eventually we found that haproxy was still listening on the actual VIP IP:

```
# head -n 1 /etc/haproxy/haproxy.d/vs_keystone-service-api.cfg
listen keystone-service-api 10.9.55.15:5000
```

I confirmed that the VIPs were inside the vips namespace and the 169.254.123.1/30 IP existed on the lo interface. After adjusting the haproxy keystone listen address to 169.254.123.1 and restarting haproxy, I was then able to communicate with the Keystone API again.

chef-client then ran successfully and haproxy config files were updated appropriately.

**Comment by breu** [comment]
@kstev - the upgrade from 4.1.2 changed the vips into the namespace, so run on the first node until success..

**Comment by breu** [comment]
hey @kstev - did you really mean 4.1.5?  We haven't released 4.1.5 yet!

**Comment by breu** [comment]
ok - spoke privately.  this is a known issue so closing the issue.
